core/src/bitmap/bitmap_data.rs

```diff
@@ -271,14 +271,18 @@ mod wrapper {
         // Provides access to the underlying `BitmapData`.
         // This should only be used when you will be overwriting the entire
         // `pixels` vec without reading from it. Cancels any in-progress GPU -> CPU sync.
-        pub fn overwrite_cpu_pixels(
+        // If the CPU pixels are dirty, syncs them to the GPU.
+        pub fn overwrite_cpu_pixels_from_gpu(
```

Comment:
note: this desyncs the comment in `struct BitmapDataWrapper`.

Reply:
I updated the comment